components/availability/src/Availability.svelte

```diff
@@ -1684,87 +1218,21 @@
                     startTime: new Date(slot.start_time).toLocaleString(),
                   }}
                   class="slot {slot.selectionStatus} {slot.availability}"
-                  class:pending={slot.selectionPending}
-                  class:hovering={slot.hovering}
+                  class:pending={event_to_hover && slot.selectionPending}
                   data-start-time={new Date(slot.start_time).toLocaleString()}
                   data-end-time={new Date(slot.end_time).toLocaleString()}
-                  on:mousedown={(event) => {
-                    if (!touchPriority)
-                      handleSlotInteractionStart({ event, slot, day });
-                  }}
-                  on:mouseenter={(event) => {
-                    if (!touchPriority) handleSlotHover({ event, slot, day });
-                  }}
-                  on:mouseleave={() => (slot.hovering = false)}
-                  on:mouseup={() => {
-                    if (!touchPriority && mouseIsDown)
-                      handleSlotInteractionEnd(day);
-                  }}
-                  on:keypress={(e) => {
-                    if (e.code === "Space" || e.code === "Enter") {
-                      startDrag(slot, day);
-                      tick().then(() => endDrag(day));
-                    }
+                  on:click={() => {
+                    handleSlotInteractionStart(slot);
```

Comment:
Is mobile behaviour just selecting a pre-defined event duration now? Assuming that `touching` a selected event again will de-select it then.

Reply:
Yes, it works exactly the same as consecutive booking did previously. The event duration is defined by the `events` prop (either passed in or set by the manifest), so clicking / touching a timeslot will attempt to select it and nearby timeslots.